loader_hub/memos/base.py

```diff
@@ -0,0 +1,54 @@
+"""Simple Reader for Memos"""
+
+from typing import List, Optional, Any
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class MemosReader(BaseReader):
+    """Memos reader.
+
+    Reads content from an Memos.
+
+    """
+
+    def __init__(self, host: str = "https://demo.usememos.com/") -> None:
+        """Init params."""
+        
+        self._memoUrl = host + "api/memo"
```

Comment:
Can we use `urljoin` here? It accounts for extra/missing slashes: https://docs.python.org/3/library/urllib.parse.html